src/cascadia/WindowsTerminal/AppHost.cpp

```diff
@@ -14,6 +18,32 @@ using namespace winrt::Windows::Foundation::Numerics;
 using namespace ::Microsoft::Console;
 using namespace ::Microsoft::Console::Types;
 
+// TODO: Before finishing PR, move this to a shared place (with main.cpp), and
+// polish the dialog a bit
+// Routine Description:
+// - Retrieves the string resource from the current module with the given ID
+//   from the resources files. See resource.h and the .rc definitions for valid IDs.
+// Arguments:
+// - id - Resource ID
+// Return Value:
+// - String resource retrieved from that ID.
+static std::wstring GetStringResource(const UINT id)

```

Comment:
lol would've been easier to drop the static and put it in a .h file or just def the prototype here :smile: